### DISCUSSION TITLE
502 Bad Gateway error while converting H5P into SCORM

### DISCUSSION THREAD

**Discussion by franklinselvaraj1985** [WeblateOrg/weblate#4809] [General]
'502 Bad Gatewa'y error while converting H5P into SCORM in https://translate-h5p.tk/scorm-h5p/

Few .H5P files get uploaded and converts into SCORM successfully but some others not. For example, a file of 22.2 MB works fine but not the one which is 27.6 MB.

Is there any threshold limit (size) to upload a .H5P file? Or is there any issue with the file converter and been reported already?

Steps to reproduce:
1. Login https://translate-h5p.tk/scorm-h5p/
2. Click 'Browse' and upload a .H5P file 
3. Click 'Package'
4. Error pops up as 502 Bad Gateway

Thanks,
Franklin

**Comment by nijel** [comment]
I think you've chosen wrong repository to report this, Weblate is not anyhow affiliated with that site.

**Comment by franklinselvaraj1985** [comment]
> I think you've chosen wrong repository to report this, Weblate is not anyhow affiliated with that site.

Thank you and apologies for the trouble. May I know where to report this bug, any guidance would be helpful for us.

**Comment by nijel** [comment]
https://github.com/sr258/scorm-h5p-wrapper seems to be the right repo, it's linked from the page you've mentioned.
